### PR TITLE
GS-66: Create feedback form

### PR DESCRIPTION
**Pull request by RobbyGardner**
Create feedback form for feedback subpage

Screen Shot 2022-08-11 at 1 16 35 PM
.

### CODE REVIEW

File: components/FeedbackForm/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  margin: 12px 0 25px;
+  height: 200px;
+  border-width: 1px;
+  border-radius: 4px;
+  padding-left: 8px;
+  padding-top: 9px;
+  font-family: Arial;
+}
+
+/* .subjectField {
```
Comment (gochoa1088) [inline components/FeedbackForm/styles.module.css:27]: clean up comments

File: pages/settings/feedback/index.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-const Feedback = () => {
+import styles from "./styles.module.css";
+
+const FeedbackPage = () => {
   return (
-    <div>
+    <div className={styles.container}>
       <Heading1>Feedback</Heading1>
+      <p>Send us your feedback!</p>
+      <FeedbackForm></FeedbackForm>
```
Comment (gochoa1088) [inline pages/settings/feedback/index.js:11]: can use the shorthand of component injection

File: components/FeedbackForm/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import styles from "./styles.module.css";
+
+const FeedbackForm = () => {
+  return (
+    <form className={styles.container}>
+      <label>Subject</label>
```
Comment (gochoa1088) [inline components/FeedbackForm/index.js:6]: for forms, labels usually wrap the element we are trying to label

https://reactjs.org/docs/forms.html

**Review by gochoa1088 — CHANGES_REQUESTED** [review]

**Review by gochoa1088 — APPROVED** [review]
nice job linking labels to their input fields

File: components/FeedbackForm/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import styles from "./styles.module.css";
+
+const FeedbackForm = () => {
+  return (
+    <form className={styles.container}>
```
Comment (joelcolucci) [inline components/FeedbackForm/index.js:5]: Nice work! Good looking form HTML.

**Pull request merged**